source/adapters/hip/device.cpp

```diff
@@ -30,6 +30,9 @@ UR_APIEXPORT ur_result_t UR_APICALL urDeviceGetInfo(ur_device_handle_t hDevice,
   static constexpr uint32_t MaxWorkItemDimensions = 3u;
 
   switch ((uint32_t)propName) {
+  case UR_DEVICE_INFO_VIRTUAL_MEMORY_SUPPORT: {
```

Comment:
Is this related?

Reply:
Not directly 👍 - will include in a separate patch